packages/back-end/src/scripts/migrate-encryption-key.ts

```diff
@@ -7,15 +7,16 @@ import { usingFileConfig } from "../init/config";
 import { ENCRYPTION_KEY, IS_CLOUD } from "../util/secrets";
 import { init } from "../init";
 import { encryptParams } from "../services/datasource";
+import { logger } from "../util/logger";
```

Comment:
The scripts directory should probably be ignored for that console rule. These are meant to be run from a terminal, so using console actually makes a lot of sense here.